tests/Mf2/ClassicMicroformatsTest.php

```diff
@@ -238,4 +238,444 @@ public function testRelBookmarkUrl() {
 
 	}
 
+
+	/**
+	 * @see http://microformats.org/wiki/microformats2-parsing-issues#uf2_children_on_backcompat_properties
+	 */
+	public function testMf2ChildrenOnBackcompatProperties()
```

Comment:
re: this test, see https://chat.indieweb.org/microformats/2017-04-25#t1493156495491000 for a discussion